src/main/asciidoc/sql/SQL-Alter-Type.adoc

```diff
@@ -11,15 +11,16 @@ The change is persistent.
 
 [source,sql]
 ----
-ALTER TYPE <type> [<attribute-name> <attribute-value>] [CUSTOM <custom-key> = <custom-value>]
+ALTER TYPE <type> [NAME <type-name>] [<attribute-name> <attribute-value>] [CUSTOM <custom-key> = <custom-value>]
```

Comment:
I think the syntax should remain the same, where "name" is just an attribute name. @gramian WDYT?

Reply:
Sure, I can change that.

Reply:
You are right that is more consistent.